src/posts/xrisk.md

```diff
@@ -5,11 +5,9 @@ description: Why AI is a risk for the future of our existence, and why we need t
 
 ## Experts are sounding the alarm
 
-AI researchers on average [believe](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/) there's a 14% chance that once we build a superintelligent AI, it will lead to "very bad outcomes (e.g. human extinction)".
+The average AI researcher [believes](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/) there's a 5% chance that once we build a superintelligent AI, it will lead to "very bad outcomes (e.g. human extinction)".
```

Comment:
Using median + using the word average is misleading (see above)

Reply:
Reverted.